### PR TITLE
No moab cmds for VUB

### PR DESCRIPTION
**Pull request by smoors**
remove showq, showstart, and checkjob (similar to UGent)

### CODE REVIEW

File: intro-HPC/ch_best_practices.tex [hunk 1]
```diff
@@ -37,9 +37,14 @@ \section{General Best Practices}
     have your default environment, so don't forget to load the software with
     \verb|module load|.
 
-  \ifgent\else\item  In case your job not running, use ``checkjob''.  It will show why your
+  \ifgent
+  \else
+  \ifbrussel
```
Comment (boegel) [inline intro-HPC/ch_best_practices.tex]: @smoors missing `else` here, unless `checkjob` works at VUB?

**Comment by boegel** [comment]
@smoors You'll also need to sync with latest `master` to avoid conflicts (lots of PRs are being merged these days thanks to @redfast00's work)

File: intro-HPC/ch_best_practices.tex (hunk 1 above)
Comment (smoors) [reply intro-HPC/ch_best_practices.tex]: @boegel should be fixed now.

**Comment by boegel** [comment]
Thanks @smoors!

**Pull request merged**